fontbakery-check-ttf.py

```diff
@@ -1724,30 +1725,16 @@ def check_bit_entry(font, table, attr, expected, bitmask, bitname):
 
     # ----------------------------------------------------
     fb.new_check("Checking with ftxvalidator")
-    KNOWN_GOOD_OUTPUT = ''\
-                        '<?xml version="1.0" encoding="UTF-8"?>\n'\
-                        '<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"'\
-                        ' "http://www.apple.com/DTDs/PropertyList-1.0.dtd">\n'\
-                        '<plist version="1.0">\n'\
-                        '<dict>\n'\
-                        '    <key>kATSFontTestResultKey</key>\n'\
-                        '    <array>\n'\
-                        '        <string>'\
-                        'kATSFontTestSeverityInformation</string>\n'\
-                        '        <string>'\
-                        'kATSFontTestSeverityMinorError</string>\n'\
-                        '    </array>\n'\
-                        '</dict>\n'\
-                        '</plist>'
-
     try:
       ftx_cmd = ["ftxvalidator",
-                 "-r",  # Generate a full report
```

Comment:
The new test will work with or without this flag. Without it, the returned string is much shorter.